tests/unit/test_ansible_inventory.py

```diff
@@ -118,3 +129,64 @@ def test_invalid_layouts(self, layout, db, metadata):
         with pytest.raises(ConfigError) as excinfo:
             ans_inv.create_inventory()
         assert "dictionary" in str(excinfo.value)
+
+    @pytest.mark.parametrize(
```

Comment:
I don't think we need to parametrize this test on layout. Layout defines have groups are nested in each other this is not related to the tested feature - host attributes.